### ISSUE TITLE
Board diversity

### ISSUE THREAD

**Issue by rougier** [ReScience/ReScience#102]
The current editorial board is not very diverse in its composition and I think we should work on that. Also, some editors have been with the journal from its very beginning and we could also have an option to propose to "retire" (if they want), maybe with an alumni section to recognize their role. It's a bit different for reviewers because this is on a voluntary basis (reviewers register themselves) and we cannot control the composition. 

In fact, maybe we need to have a public document (to be written) explaining how the board is made and/or to have public campaigns for recruiting new editors (like JOSS or JOSE). I think @labarba may have a useful experience on the matter.

**Comment by khinsen** [comment]
More generally, we should consider adopting some guidelines for the governance of ReScience C (avoiding bureaucracy of course, which is why I say "guidelines" rather than "rules"). We have been operating in pioneer mode until now, which is what it takes to get an experiment off the ground, but it's probably not a good mode of operation in the long run.

Given that I have no idea of which governance structures work for publisher-free journals, I'd be happy to take inspiration from JOSS, JOSE, and whoever else is working in this space.

**Comment by oliviaguest** [comment]
A CoC might be good. And the related discussion around how we detect and remove potential bad actors.

**Comment by rougier** [comment]
I'm not too fond of CoC.

**Comment by labarba** [comment]
JOSS asks authors and reviewers to [adhere to a CoC](https://joss.theoj.org/about#code_of_conduct). [NumFOCUS projects](https://numfocus.org/projects-overview) are required to have a Code of Conduct (CoC), and the Open Journals is a NumFOCUS project. If ReScience joins TheOJ, my guess is that at least adherence to the same CoC as JOSS will be required.

**Comment by oliviaguest** [comment]
What's wrong with guidelines for conduct?

**Comment by khinsen** [comment]
@oliviaguest Speaking strictly for myself: CoCs are such a divisive topic that the very proposition to introduce them can provoke the kind of harm they aim to prevent. Which for me means that that we don't really understand yet how to manage online communities. One obvious difficulty is where to place a CoC on the scale from "guideline" to "enforcable rule". And the more it moves towards enforcable rules, the more the issue of actually enforcing the rules needs to be solved, which most communities seem to avoid carefully.

In real life, our laws and regulations are the result of centuries of evolution. CoCs are still at the tinkering stage.

Personally, I am fine with guidelines, but I'd avoid introducing anything stricter before there is a real good reason to do so. ReScience differs from most FOSS communities in having a much higher barrier to entry: no matter which role you take, you need years of study to reach the expertise required, and invest a serious effort for your contribution to ReScience. I'd expect this to suffice to keep the average crank at a distance.

**Comment by ctb** [comment]
I would be dismayed if the official position of ReScience is that we don't need a CoC. Some kind of guidelines and enforcement of those guidelines are core aspects of any formal community.

It's a pretty well trodden space and I'd point you at the Carpentries and the Python Software Foundation as two communities that have put a lot of time and effort into thinking through both the CoC terminology and the reporting and enforcement processes.

This is distinct from @labarba point that if we want to join TheOJ we'd have to buy into theirs.

Last but not least, we generally only count the people that _do_ show up to a community, but there's a survivorship bias in that a diverse audience will generally not show up to a community that has avoided the question of how to treat foreseeable kinds of misconduct by its members. And we don't count the people that look at the project and turn away.

**Comment by oliviaguest** [comment]
We surely need guidelines for behaviour that are openly accessible and transparent, like a CoC. Otherwise, the rules are left implicit and enforceable only as a function of the willpower and current opinion each person has. That seems way more open to abuse than a set of ideas on how to behave we can all openly and explicitly discuss.

**Comment by khinsen** [comment]
@ctb Again speaking only for myself: My position is not that ReScience won't even need a CoC, but that (1) we have done pretty well without one so far and (2) I have no idea which type of CoC would be best for us in the long run.

Also, survivor bias goes both ways. Communities that do adopt a CoC don't count the people who are turned away by that CoC. And those are not all potential harassers that you do want to keep at a distance. It's also people who fear the deleterious effects of a necessarily imperfect law enforcement system.

That said, I am all for learning from other communities, but even that is a hard problem.

**Comment by oliviaguest** [comment]
Sure, I agree that you should keep thinking about it. Here's an example of us discussing CoC-like issues (although not about behaviour) just fine: https://github.com/ReScience/ReScience/issues/101; so I have faith in us being able to do it. 😊

**Comment by khinsen** [comment]
For information, here is the JOSS CoC: https://github.com/openjournals/joss/blob/master/CODE_OF_CONDUCT.md

It refers to "project maintainers" in several places, as the people responsible for enforcing the CoC. But I can't find a definition of who those project maintainers are. Perhaps the editorial board, or some subset of it?

**Comment by rougier** [comment]
Maybe I should have add the precision that I was speaking for myself. If ReScience C joins the open journals, we'll adhere to the CoC.

**Comment by oliviaguest** [comment]
@labarba knows more than me, but I think it's one of the roles each person takes within a project (not a person working for the journal). So in the same way you have a person who writes the manuscript, you have a "project maintainer". Please correct me if I am wrong though, of course! 😅

**Comment by khinsen** [comment]
Can't say if you are right or wrong, but I wonder how many people actually read the JOSS CoC, if a basic issue such as who is assigned responsibilities isn't clear. Full disclosure: as a JOSS reviewer, I have accepted the CoC without reading it, like I do with other legal documents of the Internet age.

**Comment by oliviaguest** [comment]
Is it actually about reading it (of course useful and maybe needed often) but more about it being there *when it is needed*?

**Comment by khinsen** [comment]
When it's needed it's too late to proofread and fix it. And that's a symptom of the problem I see with the current CoC fashion. If and when CoCs are actually needed, people will realize that setting up a formal governance and jurisdiction system that works is enormously more complicated than writing a CoC. When you cross the borderline from mere guidelines to enforcable rules, you need not only rules but also institutions that evolve and enforce them. And those institutions need their own rules to prevent abuse at the meta-level.

**Comment by oliviaguest** [comment]
Ah, we have a very different philosophy. I see it as a constant dialogue.

**Comment by rougier** [comment]
Maybe we should open a new issue on CoC matter because the initial topic of the issue was about the limited board's diversity and how to fix that.

**Comment by oliviaguest** [comment]
Sure, although... I think having a clear CoC and other transparent documentation is part of it.

**Comment by rougier** [comment]
Yes, but since we intend to move to theoj.org, is it worth writing one for the transition period?

**Comment by oliviaguest** [comment]
Fair point!

**Comment by broukema** [comment]
> Ah, we have a very different philosophy. I see it as a constant dialogue.

We obtained a consensus on a CoC in both of the most recent workshops that I organised, but to some degree I see both Konrad and Olivia being right :). A CoC that appears to violate both habeas corpus and the presumption of innocence risks creating an authoritarian institution if it ever has to be put into practice (enforced), despite the best intentions. On the other hand, if the dialogue flexible enough, then the ideals of avoiding patriarchy and other forms of hierarchy migh, in principle, be achievable. There are empirical, quantitative Wikipedia results for the [Iron law of oligarchy][IronLaw], but I'm not convinced that the interpretation is valid.

[IronLaw]: https://en.wikipedia.org/wiki/Iron_law_of_oligarchy%23Wikipedia
